model/inline/src/main/scala/aqua/model/inline/ArrowInliner.scala

```diff
@@ -359,28 +354,24 @@ object ArrowInliner extends Logging {
   private def prelude[S: Mangler: Arrows: Exports](
     fn: FuncArrow,
     call: CallModel,
-    oldExports: Map[String, ValueModel]
+    oldExports: Map[String, ValueModel],
+    arrows: Map[String, FuncArrow]
```

Comment:
Update function doc comment, please
Also why call it `oldExports` and not `exports`?